Python/marshal.c

```diff
@@ -1333,6 +1334,7 @@ r_object(RFILE *p)
             kwonlyargcount = (int)r_long(p);
             if (PyErr_Occurred())
                 goto code_error;
+            // XXX We no longer use nlocals.
             nlocals = (int)r_long(p);
```

Comment:
Then why not just write
```
r_long(p);  // nlocals is no longer used
```
(Also why not just remove it from the marshal format and change the magic number?)

Reply:
I was planning on doing it in the next PR where I'm changing the magic number for something else.  I don't mind doing it here though.

Reply:
That's fine then.